File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/ClientAPI/WidthHeight/WidthHeightPage.vb
Imports System.Drawing
Imports Gizmox.WebGUI.Forms.CompanionKit.Concepts.ClientAPI.WidthHeight

Namespace CompanionKit.Concepts.ClientAPI.WidthHeight

    Public Class WidthHeightPage

        Public Sub New()

            ' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()

            'Add info label to user control
            Dim mobjLabel As New Label()
            mobjLabel.Dock = DockStyle.Top
            mobjLabel.Padding = New Padding(10, 10, 0, 0)
            mobjLabel.Text = "'A' and 'D' buttons change width of TextBox, 'S' and 'W' - height."
            mobjLabel.Size = New Size(20, 55)
            mobjLabel.AutoSize = False

            Me.Controls.Add(mobjLabel)

            'Add custom textBox to user control
            Dim mobjTextBox As New MyTextBox()
            mobjTextBox.CustomStyle = "MyTextBoxSkin"
            mobjTextBox.Location = New Point(10, 60)
            mobjTextBox.Multiline = True
            mobjTextBox.Size = New Size(200, 100)

            Me.Controls.Add(mobjTextBox)

        End Sub
    End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/ClientAPI/WidthHeight/WidthHeightPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms.CompanionKit.Concepts.ClientAPI.WidthHeight;

namespace CompanionKit.Concepts.ClientAPI.WidthHeight
{
    public partial class WidthHeightPage : UserControl
    {
        public WidthHeightPage()
        {
            InitializeComponent();

            //Add info label to user control
            Label mobjLabel = new Label();
                mobjLabel.Dock = DockStyle.Top;
                mobjLabel.Padding = new Padding(10, 10, 0, 0);
                mobjLabel.Text = "'A' and 'D' buttons change width of TextBox, 'S' and 'W' - height.";
                mobjLabel.Size = new Size(20,55);
                mobjLabel.AutoSize=false;
            this.Controls.Add(mobjLabel);

            //Add custom textBox to user control
            //this one also
            MyTextBox mobjTextBox = new MyTextBox();
                mobjTextBox.CustomStyle = "MyTextBoxSkin";
                mobjTextBox.Location = new Point(10, 60);
                mobjTextBox.Multiline=true;
                mobjTextBox.Size = new Size(200,100);
            this.Controls.Add(mobjTextBox);
        }
    }
}